templates/quickstart/react/package.json

```diff
@@ -21,7 +21,7 @@
     "react": "15.4.2",
     "react-dom": "15.4.2",
     "react-router": "2.8.0",
-    "igniteui-react": "^1.2.0",
+    "igniteui-react": "^1.3.1",
```

Comment:
The latest version of `igniteui-react` is `1.3.0`. The specified version here causes the packages install to fail.